CONTRIBUTING.md

```diff
@@ -6,39 +6,46 @@ If you are interested in contributing to Micronaut and are looking for issues to
 
 ## JDK Setup
 
-Micronaut XXX currently requires JDK 8
+Micronaut XXX currently requires JDK 8.
 
 ## IDE Setup
 
 Micronaut XXX can be imported into IntelliJ IDEA by opening the `build.gradle` file.
 
 ## Docker Setup
 
-Micronaut XXX tests currently require docker to be installed.
+Micronaut XXX tests currently require Docker to be installed.
 
 ## Running Tests
 
-To run the tests use `./gradlew check`.
+To run the tests, use `./gradlew check`.
 
 ## Building Documentation
 
 The documentation sources are located at `src/main/docs/guide`.
 
-To build the documentation run `./gradlew publishGuide` or `./gradlew pG` then open `build/docs/index.html`  
+To build the documentation, run `./gradlew publishGuide` (or `./gradlew pG`), then open `build/docs/index.html`
 
-To also build the javadocs instead run `./gradlew docs`.
+To also build the Javadocs, run `./gradlew docs`.
 
 ## Working on the code base
 
-If you are working with the IntelliJ IDEA development environment, you can import the project using the Intellij Gradle Tooling ( "File / Import Project" and select the "settings.gradle" file).
+If you use IntelliJ IDEA, you can import the project using the Intellij Gradle Tooling ("File / Import Project" and selecting the "settings.gradle" file).
 
 To get a local development version of Micronaut XXX working, first run the `publishToMavenLocal` task.
 
 ```
 ./gradlew pTML
 ```
 
-You can then reference the version specified with `projectVersion` in `gradle.properties` in a test project's `build.gradle` or `pom.xml`.
+You can then reference the version specified with `projectVersion` in `gradle.properties` in a test project's `build.gradle` or `pom.xml`. If you use Gradle, add the `mavenLocal` repository (Maven automatically does this):
+
```

Comment:
`If you use Gradle` we always use Gradle for the modules :stuck_out_tongue_closed_eyes: 

The only one that doesn't is the Maven project, but it's not a fork of this one.

Reply:
these instructions are for a test project using a locally built module

Reply:
You are right, sorry for the misunderstanding :pray: